styles/background.module.css

```diff
@@ -0,0 +1,29 @@
+.logoOnJoinUs {
+    background-image: url(/britologohalf.png);
```

Comment:
You don't need 2 different images. set the dimensions using CSS

Reply:
done